### DISCUSSION TITLE
VERCEL_ENV set as staging, is it ok?

### DISCUSSION THREAD

**Discussion by mzvdev** [vercel/community#3717] [Feature Question]
### Summary

Hi,

I have read your guide about how to setup staging environment : https://vercel.com/guides/set-up-a-staging-environment-on-vercel.
I would like to know if it's okay to set: in environmet variables: **VERCEL_ENV=staging** for preview branch staging.
it works, but I'm afraid that it can create bugs.

<img width="963" alt="image" src="https://github.com/vercel/community/assets/139748787/f3052234-823a-4b68-8c35-98d386f4910e">

I need to know that I'm in a staging environment to seed my Staging database with Prisma seed.
Maybe I should do it an other way...

Kind regards,

### Steps to Reproduce

- no bugs for now.

**Comment by Divinebosst** [comment]
According to the documentation, VERCEL_ENV only accepts 3 values that is production, development and preview: https://vercel.com/docs/projects/environment-variables/system-environment-variables
